components/collector/src/quality_time_collector.py

```diff
@@ -2,18 +2,22 @@
 
 import asyncio
 import logging
+import os
+from typing import NoReturn
 
 # Make sure subclasses are registered
 import metric_collectors  # pylint: disable=unused-import # lgtm [py/unused-import]
 import source_collectors  # pylint: disable=unused-import # lgtm [py/unused-import]
 from base_collectors import Collector
 
 
-async def collect(log_level: int | None = None) -> None:
+async def collect(log_level: int | str | None = None) -> NoReturn:
```

Comment:
Is `collect` ever called with the `log_level` param as int? Can the parameter be removed in favour of the env var?

Reply:
I assumed it was, from the unit tests. But assumptions are the mother of all fuckups...